graph/tags/tags.go

```diff
@@ -16,7 +18,7 @@ func (e ErrTagInvalidFormat) Error() string {
 	return fmt.Sprintf("Illegal tag name (%s): only [A-Za-z0-9_.-] are allowed ('.' and '-' are NOT allowed in the initial), minimum 1, maximum 128 in length", e.name)
 }
 
-// ValidateTagName validates the name of a tag
+// ValidateTagName validates the name of a tag.
```

Comment:
I'll add: "It returns an error if the given name is an emtpy string. If name does not match v2.TagNameAnchoredRegexp regexp, it returns ErrTagInvalidFormat"